pynamodb/connection/base.py

```diff
@@ -351,10 +351,14 @@ def _make_api_call(self, operation_name, operation_kwargs):
 
             response = None
             try:
+                proxies = getattr(self.client._endpoint, "proxies", None)
```

Comment:
A comment regarding the change of internal behavior in `1.11.0` might be helpful here

Reply:
:+1: